src/generators/java/mekanism/generators/common/content/fusion/FusionReactorMultiblockData.java

```diff
@@ -345,42 +342,45 @@ protected int getMultiblockRedstoneLevel() {
-        double k = active ? caseWaterConductivity : 0;
+        double k = active ? MekanismGeneratorsConfig.generators.fusionWaterHeatingRatio.get() : 0;
+        double caseAirConductivity = MekanismGeneratorsConfig.generators.fusionCasingThermalConductivity.get();
         return injectionRate * MekanismGeneratorsConfig.generators.energyPerFusionFuel.get().doubleValue() / plasmaCaseConductivity *
                (plasmaCaseConductivity + k + caseAirConductivity) / (k + caseAirConductivity);
     }
 
     public double getMaxCasingTemperature(boolean active) {
-        double k = active ? caseWaterConductivity : 0;
-        return MekanismGeneratorsConfig.generators.energyPerFusionFuel.get().multiply(injectionRate).divide(k + caseAirConductivity).doubleValue();
+        double k = active ? MekanismGeneratorsConfig.generators.fusionWaterHeatingRatio.get() : 0;
+        return MekanismGeneratorsConfig.generators.energyPerFusionFuel.get().multiply(injectionRate).divide(k + MekanismGeneratorsConfig.generators.fusionCasingThermalConductivity.get()).doubleValue();
     }
 
     public double getIgnitionTemperature(boolean active) {
-        double k = active ? caseWaterConductivity : 0;
+        double k = active ? MekanismGeneratorsConfig.generators.fusionWaterHeatingRatio.get() : 0;
+        double caseAirConductivity = MekanismGeneratorsConfig.generators.fusionCasingThermalConductivity.get();
         double energyPerFusionFuel = MekanismGeneratorsConfig.generators.energyPerFusionFuel.get().doubleValue();
         return burnTemperature * energyPerFusionFuel * burnRatio * (plasmaCaseConductivity + k + caseAirConductivity) /
                (energyPerFusionFuel * burnRatio * (plasmaCaseConductivity + k + caseAirConductivity) - plasmaCaseConductivity * (k + caseAirConductivity));
     }
 
     public FloatingLong getPassiveGeneration(boolean active, boolean current) {
         double temperature = current ? getLastCaseTemp() : getMaxCasingTemperature(active);
-        return FloatingLong.create(thermocoupleEfficiency * caseAirConductivity * temperature);
+        return FloatingLong.create(MekanismGeneratorsConfig.generators.fusionThermocoupleEfficiency.get() * MekanismGeneratorsConfig.generators.fusionCasingThermalConductivity.get() * temperature);
     }
 
     public long getSteamPerTick(boolean current) {
         double temperature = current ? getLastCaseTemp() : getMaxCasingTemperature(true);
-        return MathUtils.clampToLong(HeatUtils.getSteamEnergyEfficiency() * caseWaterConductivity * temperature / HeatUtils.getWaterThermalEnthalpy());
+        return MathUtils.clampToLong(HeatUtils.getSteamEnergyEfficiency() * MekanismGeneratorsConfig.generators.fusionWaterHeatingRatio.get() * temperature / HeatUtils.getWaterThermalEnthalpy());
     }
 
     public static double getInverseConductionCoefficient() {
-        return 1 / caseAirConductivity;
+        return 1 / MekanismGeneratorsConfig.generators.fusionCasingThermalConductivity.get();
```

Comment:
I will do a closer pass tomorrow, but min value for this config option and any others that can be divided by should be something greater than zero and not zero.

Reply:
Ahh good catch. Do you typically just have them set to 0.001 or something more interesting?

Reply:
0.001 should be fine normally we use 1 but normally the cases we have it is an int not a double.

Reply:
I did a quick readthrough with an eye toward looking for div0 cases, I think it was just this configurable that needed tweaking  this way to prevent div0 cases at least in this file. (I didn't think too hard about which functions might return 0 used by other callers.)